Lib/test/test_socket.py

```diff
@@ -75,12 +94,20 @@ def _have_socket_alg():
         s.close()
     return True
 
+def _have_socket_vsock():
+    """Check whether AF_VSOCK sockets are supported on this host."""
+    if (getCid() == None):
```

Comment:
Yeah, that's what I was trying to say. Sorry for being unclear!

And +1 for renaming it to `get_cid`, thanks!